api/service/staking/service.go

```diff
@@ -5,21 +5,59 @@ import (
 	"github.com/harmony-one/harmony/p2p"
 )
 
```

Comment:
This is already added into master: https://github.com/harmony-one/harmony/pull/376

Reply:
Can you add on top of that.